app/utils.ts

```diff
@@ -45,3 +46,10 @@ export function useMatchesData(
 export function validateEmail(email: unknown): email is string {
   return typeof email === "string" && email.length > 3 && email.includes("@")
 }
+
+export const svgToDataUri = (
+  element: Parameters<typeof renderToStaticMarkup>[0]
+) => {
+  const svgString = encodeURIComponent(renderToStaticMarkup(element))
+  return `data:image/svg+xml,${svgString}`
+}
```

Comment:
I'm going to leave this in, but I'm wondering why we need this? Should have raised this in the huddle.

Reply:
I originally created this to be used in storybook stories, but I think it can now probably be removed and replaced with URLs.